scripts/buildexamples.js

```diff
@@ -7,14 +7,7 @@ var fs = require('fs');
 // Build all of the example folders.
 dirs = fs.readdirSync('examples');
 
-var cmd;
 for (var i = 0; i < dirs.length; i++) {
-  if (dirs[i].indexOf('.') !== -1) {
-    continue;
-  }
-  if (dirs[i].indexOf('node_modules') !== -1) {
-    continue;
-  }
   console.log('Building: ' + dirs[i] + '...');
```

Comment:
I've been meaning to add some better delimiters here (like a line of asterisks), want to do that in this PR?